src/modules/tests/pre-test-declarations/__tests__/pre-test-declarations.reducer.spec.ts

```diff
@@ -6,7 +6,7 @@ import {
   SignatureDataCleared,
   ClearPreTestDeclarations,
 } from '../common/pre-test-declarations.actions';
-import { PreTestDeclarations } from '@dvsa/mes-test-schema/categories/common';
+import { PreTestDeclarations } from '@dvsa/mes-test-schema/categories/AM1/index';
```

Comment:
shouldn't this be `import { PreTestDeclarations } from '@dvsa/mes-test-schema/categories/AM1`